crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -26,29 +27,62 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for GlobalpayPaymentsRequest {
             .and_then(|o| o.get("account_name"))
             .map(|o| o.to_string())
             .ok_or_else(utils::missing_field_err("connector_meta.account_name"))?;
-        let card = item.get_card()?;
-        Ok(Self {
-            account_name,
-            amount: Some(item.request.amount.to_string()),
-            currency: item.request.currency.to_string(),
-            reference: item.attempt_id.to_string(),
-            country: item.get_billing_country()?,
-            capture_mode: item.request.capture_method.map(|f| match f {
-                enums::CaptureMethod::Manual => requests::CaptureMode::Later,
-                _ => requests::CaptureMode::Auto,
+
+        match item.request.payment_method_data.clone() {
+            api::PaymentMethod::Card(ccard) => Ok(Self {
+                account_name,
+                amount: Some(item.request.amount.to_string()),
+                currency: item.request.currency.to_string(),
+                reference: item.attempt_id.to_string(),
+                country: item.get_billing_country()?,
+                capture_mode: item.request.capture_method.map(|f| match f {
+                    enums::CaptureMethod::Manual => requests::CaptureMode::Later,
+                    _ => requests::CaptureMode::Auto,
+                }),
+                payment_method: requests::PaymentMethod {
+                    card: Some(requests::Card {
+                        number: ccard.get_card_number(),
+                        expiry_month: ccard.get_card_expiry_month(),
+                        expiry_year: ccard.get_card_expiry_year_2_digit(),
+                        cvv: ccard.get_card_cvc(),
+                        ..Default::default()
```

Comment:
Please do not use default in these cases. If any field is optional then provide it as none ( helps in debugging ) or better to move it to a different struct where it will be mandatory and use serde(flatten) here.